modules/png-extraction/ImageExtractor.py

```diff
@@ -24,26 +24,26 @@
 import pathlib
 configs = {}
 
-def initialize_Values(config_values):
+def initialize_config_and_execute(config_values):
     global configs
     configs = config_values
     # Applying checks for paths
     
-    p1 = pathlib.PureWindowsPath(configs['dcmFolder'])
+    p1 = pathlib.PurePath(configs['DICOMHome'])
     dicom_home = p1.as_posix() #the folder containing your dicom files
 
-    p2 = pathlib.PureWindowsPath(configs['outputFolder'])
+    p2 = pathlib.PurePath(configs['OutputDirectory'])
```

Comment:
This seems like you are reverting some commits from the dev branch.

Please make sure to take a git pull from dev of Niffler to your fork before a pull request.